website/docs/docs/develop/develop-in-the-cloud.md

```diff
@@ -0,0 +1,249 @@
+- Cold start - This is the process of starting a new develop session, which will be available for you for three hours. The environment automatically turns off three hours after the last activity with the rpc server. This includes compile, preview, or any dbt invocation, however, it *does not* include editing and saving a file.
+- Hot start -  This is the state of resuming an existing or active develop session within 3 hours of the last activity. 
+    
+
+</td><td>
+    
+
+The Cloud IDE needs explicit action to save your changes and there are three ways your work is stored:
+
+- Unsaved, local code - Any code you write is automatically available from your browser’s storage. You can see your changes but will lose them if you switch branches or browsers (another device or browser).
+- Saved but uncommitted code - When you save a file, the data gets stored in your local storage (EFS storage). If you switch branches but don’t *commit* your saved changes, you will lose your changes.
+- Committed code - This is stored in the branch with git provider and you are able to check out other (remote) branches. 
+
+
+</td></tr> </table>
+
+### New Cloud IDE beta features
+
+**Format**
+You can format/preview/compile or build your dbt project, as well as see the DAG. The new **Format** button formats your file and is powered by [sqlfmt](http://sqlfmt.com/).
+
+**Multiple selections** 
+
+You can make multiple selections for quick and simultaneous edits. The below commands are a common way to add more cursors and allow you to insert cursors below or above with ease.
+
+- Option-Command-Down arrow
+- Option-Command-Up arrow
+- Press Option and click on an area
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/multi-selector.gif" title="Multiple Selections"/>
+
+**Drag and drop** 
+
+You can also drag and drop files located in the file explorer. Use the file breadcrumb on the top of the IDE for quick, linear navigation. You can access adjacent files in the same file by right clicking on the breadcrumb file.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/file-explorer.png" title="File Explorer"/> 
+
+**Organize tabs**
+
+You can move your tabs around to reorganize your work in the IDE. You can also right click on a tab to view and select a list of actions to take.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/tab-options.png" title="Tab options"/>
+
+**Run projects**
+
+You can also *build*, *run* *and test* dbt projects directly in the Cloud IDE with a click of a button, using the ‘Build’ feature. You can also use dbt's [rich model selection syntax](https://docs.getdbt.com/reference/node-selection/syntax) to [run dbt commands](https://docs.getdbt.com/reference/dbt-commands) directly within dbt Cloud.
+
+The IDE updates in real-time as models, tests, seeds, and operations are run. If a model or test fails, you can review  the logs to find and fix the issue.
```

Comment:
```suggestion
The IDE updates in real time as models, tests, seeds, and operations are run. If a model or test fails, you can review  the logs to find and fix the issue.
```

Reply:
a hyphen is used when it's a compound modifier like in "real-time operating system" or a "real-time payments"